boards/arm/96b_meerkat96/96b_meerkat96-pinctrl.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nxp/nxp_imx/mimx7d-pinctrl.dtsi>
+
+
+&pinctrl {
+	pinmux_uart1: pinmux_uart1 {
```

Comment:
suggestion: better name nodes `periph_state`, e.g. `uart1_default`.

Reply:
Changed node names to match this pattern